Source/GTMSessionFetcher.m

```diff
@@ -3462,6 +3480,7 @@ + (GTM_NULLABLE GTMSessionFetcherSystemCompletionHandler)systemCompletionHandler
             sendProgressBlock = _sendProgressBlock,
             willCacheURLResponseBlock = _willCacheURLResponseBlock,
             retryBlock = _retryBlock,
+            metricsCollectionBlock = _metricsCollectionBlock,
```

Comment:
Please set self.metricsCollectionBlock = nil inside releaseCallbacks.

Reply:
It's already there in this PR: https://github.com/google/gtm-session-fetcher/pull/188/files#diff-590105fbe264a2024b0ac16eec191826R1799